app/screens/settings/config.ts

```diff
@@ -16,6 +17,18 @@ export const getSaveButton = (buttonId: string, intl: IntlShape, color: string)
     ...typography('Body', 100, 'SemiBold'),
 });
 
+export const gotoSettingsScreen = (screen: string, title: string) => {
+    const passProps = {};
+    const options = {
+        topBar: {
+            backButton: {
+                popStackOnPress: false,
```

Comment:
why are we doing this?

Reply:
To hijack the software back button and run a callback that will save the newly changed value for each setting - https://wix.github.io/react-native-navigation/docs/customBackNavigation#handling-the-software-back-button